locales/en.json

```diff
@@ -141,6 +141,26 @@
       "addButton": "Add payment method",
       "addDescription": "You can add your payment methods (e.g. credit cards, bank accounts) so that you don't have to re-add them from scratch for every transaction"
     },
+    "firstTransactionSummary":{
+      "header": "Transaction notice summary",
+      "title": "Transaction notice",
+      "amount":"Amount EUR",
+      "expireDate":"Expire date",
+      "tranche":"Tranche",
+      "entity":"Entity:",	
+      "info":"More info at:",
+      "tel":"Tel:",
+      "email":"Email:",
+      "PEC":"PEC:",
+      "recipient":"Recipient:",
+      "object":"Transaction object:",
+      "cbillCode":"CBILL code:",
+      "iuv":"IUV:",
+      "entityCode2":"Entity/Company code:",
+      "updatedAmount":"Updated amount",
+      "updateInfo":"The pagoPa system updates automatically the amount to ensure you pay the exact amount and to avoid you arrears or addictional interests ",
```

Comment:
arrears -> errors ?

Reply:
(arrears refers to owed money)
addictional -> additional
I would rephrase the second part of the statement to "to make sure you pay the expected amount"

Reply:
arrears stands also for "interessi di mora". If you prefer we can also rephrase